custom_components/auto_backup/translations/sk.json

```
@@ -0,0 +1,25 @@
+{
+    "title": "Auto záloha",
```

Typically you don't translate the name, right? That's at least what the other translations have done, and if we did we'd translate the whole name.
```suggestion
    "title": "Auto Backup",
```